Starcraft2Client.py

```diff
@@ -38,6 +38,30 @@
 class StarcraftClientProcessor(ClientCommandProcessor):
     ctx: SC2Context
 
+    def _cmd_difficulty(self, difficulty: str = "") -> bool:
+        """Overrides the current difficulty set for the seed.  Takes the argument casual, normal, hard, or brutal"""
+        options = difficulty.split()
+        num_options = len(options)
+        set_difficulty = True
+
+        if num_options > 0:
+            if options[0] == "Casual" or "casual":
```

Comment:
you can do new_text = text.lower() to get all lowercase, to ignore case.

Reply:
Thanks, will implement.